scenarios/csharp/dotnet/captioning/Program.cs

```diff
@@ -0,0 +1,417 @@
+/*
+Dependencies
```

Comment:
Categorizing pieces of code like the `using` statements seems a little too guided; I'd generally suggest we stay away from adding documentation for anything but our product usage and *significantly* non-intuitive non-product stuff--standard `using` boilerplate is table stakes enough that it needs no intro (and if it does, we've got other problems)